templates/index.html

```diff
@@ -1,18 +1,69 @@
 {% extends "layout.html" %}
-{% block title %}To-Do App{% endblock %}
+{% block title %}Jason B To-Do App{% endblock %}
 
 {% block content %}
-  <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
+  <div class="jumbotron" style="background-color:DodgerBlue;color:White;">
+    <h1 class="display-4">My To-Do App</h1>
+      <hr class="my-4">
+      <p class="lead">This is my To-Do app, enjoy...</p>
   </div>
 
+    <div class="row" style="border:2px solid DodgerBlue;">
+        <div class="col-md-12" style="margin-top: 20px; margin-bottom: 20px;">
+            <h2>Add a New To-Do Item</h2>
+            <form action="/create" method="POST">
+                <div class="form-inline">
+                    <div class="form-group" style="flex: 1">
+                        <label class="col-sm-1" for="add_todo">Title:</label>
+                        <input style="flex: 1" class="form-control" type="text" id="add_todo" name="add_todo" maxlength="64" placeholder="New to-do">
+                    </div>
+                    <div>
+                        <button class="btn btn-dark" type="submit">Add</button>
+                    </div>
+                </div>
+            </form>
+        </div>
+    </div>
+
   <div class="row">
-    <div class="col-md-12">
-      <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-      </ul>
+    <div class="col-md-12" style="margin-top: 40px;">
+      <h2 id="todo_table">Your To-Do List</h2>
+        <p>This is a list of your to-do items, check them off once completed or delete as required</p>
+        <form action="/update" method="POST">
+            <div class="form-group">
+                <table class="table table-striped" style="width: 100%">
+                    <thead>
+                    <tr>
+                        <th>Title</th>
+                        <th>Status</th>
+                        <th>Action</th>
+                    </tr>
+                    </thead>
+                    {% for item in items %}
+                        <tr>
+                            <td>{{item.title}}</td>
+                            <td>{{item.status}}</td>
+                            <td>
+                                <select class="custom-select" name="{{ item.id }}">
+                                    {% if item.status == 'Completed' %}
+                                    <option value="Not Started" name="{{ item.id }}">Not Started</option>
```

Comment:
These 2 lines should be indented one level in to help show they are part of the `{% if %}` block